PHPCompatibility/Sniffs/Syntax/NewDynamicAccessToStaticSniff.php

```diff
@@ -24,7 +24,7 @@
  *
  * PHP version 5.3
  *
- * @link https://www.php.net/manual/en/migration53.new-features.php
+ * @link https://php-legacy-docs.zend.com/manual/php5/en/migration53.new-features
```

Comment:
Does Zend offer language agnostic URLs ?

We don't use those consistently, but try to remember to use the language agnostic URLs from php.net, so just wondering if that's available for the legacy docs too.

Reply:
No, it doesn't. It was done this way to help with some redirect mapping back when 404 auto-redirect was on the table. I don't remember exactly.